gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -120,7 +120,17 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
-
+    parser.add_argument(
+        '--output_avro_path',
```

Comment:
Are we reading this just to deprecate it? If so, sg.

Reply:
Yes, I should have not deleted this in my earlier PR.